docs/reference/commandline/events.md

```diff
@@ -30,7 +30,7 @@ Options:
 
 Docker containers report the following events:
 
-    attach, commit, copy, create, destroy, detach, die, exec_create, exec_detach, exec_start, export, health_status, kill, oom, pause, rename, resize, restart, start, stop, top, unpause, update
+    archive-path, attach, checkpoint, commit, copy, create, destroy, detach, die, exec_create, exec_detach, exec_start, export, extract-to-dir, health_status, kill, oom, pause, rename, resize, restart, start, stop, top, unpause, update
```

Comment:
Thanks for your detailed explanation. @jlhawn  Happy to see the original thought on implementation.
Then though it is work one and half years ago, what do you think is a better way to continue the following things, revert to the right naming thing or keep this in documents? @thaJeztah @jlhawn